index.js

```diff
@@ -1,14 +1,22 @@
 "use strict"
 
-var createMachine = function(reducersObject) {
+var defaultGetStatus = function (state) {
+    return state.status
+}
+
+var defaultSetStatus = function (state, status) {
+    return Object.assign({}, state, {'status': status})
+}
+
+var createMachine = function(reducersObject, getStatus=defaultGetStatus, setStatus=defaultSetStatus) {
```

Comment:
So clean. I like how you improved the functionality while making minimal tweaks to the code and taking advantage of default params.